### DISCUSSION TITLE
how to select a default value in q-select?

### DISCUSSION THREAD

**Discussion by emdeweb** [quasarframework/quasar#10024] [General - Components / Directives / etc]
I  need my q-select to select the "name" depending on the previously assigned "id". Currently the input shows me the "id" in number and not the name to which it belongs.

`<q-select
  class="text-uppercase"
  v-model="model"
  outlined
  dense
  use-input
  input-debounce="0"
  label="Marcas"
  :options="options"
  option-label="name"
  option-value="id"
  emit-value
  map-options
  @filter="filterFn"
  @update:model-value="test()"
>
  <template v-slot:no-option>
    <q-item>
      <q-item-section class="text-grey"> No results </q-item-section>
    </q-item>
  </template>
</q-select>`

**Example I would like the name that has the id: 12 to be shown loaded in the q-select.**

`const model = ref(12);
const options = ref([]);

const filterFn = (val, update) => {
  if (val === "") {
    update(() => {
      options.value = tableData.value;
    });
    return;
  }

  update(() => {
    const needle = val.toLowerCase();
    options.value = tableData.value.filter((v) =>
      v.name.toLowerCase().includes(needle)
    );
  });
};`

**Comment by ontwikkelfabriek** [comment]
@emdeweb 

Please create a codepen.io to demo your problem for better support. If you have to post a code listing do so with the correct **code markup**, otherwise people are not going to spend time to solve your question.
